acceptance-tests/tests/src/test/java/org/hyperledger/besu/tests/acceptance/multitenancy/MultiTenancyAcceptanceTest.java

```diff
@@ -0,0 +1,240 @@
+  private final Account sender = accounts.createAccount("account1");
+  private final Address senderAddress =
+      Address.wrap(Bytes.fromHexString(accounts.getPrimaryBenefactor().getAddress()));
+
+  @Rule public WireMockRule wireMockRule = new WireMockRule(options().port(ENCLAVE_PORT));
+
+  @Before
+  public void setUp() throws Exception {
+    final ClusterConfiguration clusterConfiguration =
+        new ClusterConfigurationBuilder().awaitPeerDiscovery(false).build();
+    multiTenancyCluster = new Cluster(clusterConfiguration, net);
+    node =
+        besu.createNodeWithMultiTenancy(
+            "node1",
+            "http://127.0.0.1:" + ENCLAVE_PORT,
+            "authentication/auth_priv.toml",
+            "authentication/auth_pub_key",
+            "authentication/auth_priv_key");
+    multiTenancyCluster.start(node);
+    token = node.execute(permissioningTransactions.createSuccessfulLogin("user", "pegasys"));
+  }
+
+  @Test
+  public void getPrivacyPrecompileAddressShouldReturnExpectedAddress() {
+    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+    node.verify(priv.privGetPrivacyPrecompileAddressSuccess(DEFAULT_PRIVACY));
+  }
+
+  @Test
+  public void shouldGetPrivateTransaction() throws JsonProcessingException {
+    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+
+    final String base64SignedPrivateTransactionRLP =
+        "+MyAAYJSCJQJXnuupqbHxMLf65d++sMmr1Uth6D//////////////////////////////////////////4AboEi1W/qRWseVxDGXjYpqmStijVV9pf91mzB9SVo2ZJNToB//0xCsdD83HeO59/nLVsCyitQ2AbSrlJ9T+qB70sgEoANWlbTMSwlB5gVR16Gc8wYD21v8I+WsQ6VvV/JfdUhqoA8gDohf8p6XPiV2tmABgdGworUpTjDZvkoZgf+zOguMinJlc3RyaWN0ZWQ=";
+    // privateFrom value from above transaction
+    final String privateFrom = "A1aVtMxLCUHmBVHXoZzzBgPbW/wj5axDpW9X8l91SGo=";
+    final Hash transactionHash = node.execute(accountTransactions.createTransfer(sender, 50));
+    multiTenancyCluster.verify(sender.balanceEquals(50));
+
+    final String receiveResponse =
+        mapper.writeValueAsString(
+            new ReceiveResponse(
+                base64SignedPrivateTransactionRLP.getBytes(UTF_8), PRIVACY_GROUP_ID, "senderKey"));
+
+    stubFor(post("/receive").willReturn(ok(receiveResponse)));
+
+    node.verify(priv.privGetPrivateTransactionSuccess(transactionHash.toString(), privateFrom));
+  }
+
+  @Test
+  public void createPrivacyGroupSuccessShouldReturnNewId() throws JsonProcessingException {
+    final String createGroupResponse = mapper.writeValueAsString(testPrivacyGroup(emptyList()));
+
+    final CreatePrivacyGroupParameter params =
+        new CreatePrivacyGroupParameter(
+            List.of(KEY1, KEY2, KEY3), "GroupName", "Group description.");
+    stubFor(post("/createPrivacyGroup").willReturn(ok(createGroupResponse)));
+
+    node.useAuthenticationTokenInHeaderForJsonRpc(token);
+    node.verify(priv.privCreatePrivacyGroupSuccess(params, PRIVACY_GROUP_ID));
```

Comment:
Think this would be nicer without the intermediary CreatePrivacyGroupParameter object with just passing through the keys, name and description as 3 params to the privCreatePrivacyGroupSuccess condition